File: ecobertura.ui/test/ecobertura/ui/annotation/NoValidAnnotationModelExceptionTest.java
package ecobertura.ui.annotation;

import static org.junit.Assert.*;


import org.junit.Before;
import org.junit.Test;

import ecobertura.ui.annotation.NoValidAnnotationModelException;

public class NoValidAnnotationModelExceptionTest {

	Throwable t;
	
	@Before
	public void setUp() {
		t = new NoValidAnnotationModelException("message");
	}
	
	@Test
	public void testGetMessage() {
		assertEquals("message", t.getMessage());
	}

}

File: ecobertura.ui/src/test/scala/ecobertura/ui/annotation/NoValidAnnotationModelExceptionTest.scala
package ecobertura.ui.annotation

import org.junit._
import org.junit.Assert._

class NoValidAnnotationModelExceptionTest {
	var t: Throwable = null
	
	@Before
	def setUp = {
		t = new NoValidAnnotationModelException("message")
	}
	
	@Test
	def testGetMessage = {
		assertEquals("message", t.getMessage)
	}
}